src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -101,7 +98,6 @@ public String getTestConfigItem(String item) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes